pkg/athena/macros.go

```diff
@@ -114,10 +114,28 @@ func macroTimeFrom(query *sqlds.Query, args []string) (string, error) {
 
 }
 
+func macroRawTimeFrom(query *sqlds.Query, args []string) (string, error) {
+	format := timestampFormat
+	if len(args) == 1 && args[0] != "" {
+		format = args[0]
+	}
+
+	return fmt.Sprintf("format_datetime(TIMESTAMP '%s',%s)", query.TimeRange.From.UTC().Format(goTimestampFormat), format), nil
```

Comment:
the goal of this was to return the `raw` format (this is, without applying any formatting). What we should return here is just a string of the time in whatever the format that is provided (with a default format). For example  `rawTimeFrom('yyyy/MM/dd/HH/mm/ss')` should return `2022/03/25/16/57/10/59`.

Reply:
Is there a canonical Go way of translating a Joda DateTime format into Go's layout format? I couldn't find one in [the Go docs](https://pkg.go.dev/time#Time.Format) so was using [`format_datetime`](https://prestodb.io/docs/current/functions/datetime.html#format_datetime) to perform the formatting.